test/python/tools/jupyter/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Tests for Jupyter-related functionality."""
```

Comment:
This seems a bit absurd we need a docstring for an empty file? Also the copyright header doesn't really apply because there is no "source code" to license.

Reply:
Technically it is not "a docstring for an empty file" - it is the docstring for the _package_.

But on a more practical tone, it's mostly about sticking to the current convention we have in place and being consistent - I'm personally happy to reevaluate it as a whole if needed, but seeing that it was mostly introduced at #550 and that it enters the legal territory, I'd err on the side of caution in these cases.